.github/workflows/ci-macos.yml

```diff
@@ -14,11 +14,13 @@ jobs:
 
       - name: Install system dependencies
         run: |
-          brew install gnu-time opam gtksourceview3 adwaita-icon-theme expat libxml2 pkg-config
+          brew unlink pkg-config
+          brew install gnu-time opam gtksourceview3 adwaita-icon-theme expat libxml2 pkgconf
+          brew unlink pkgconf
+          brew link pkg-config
```

Comment:
this seems to be working even though it's a strange back-and-forth

note that the image comes with pkg-config preinstalled AFAICT

History:
With the previous `install pkg-config` we get error https://github.com/coq/coq/actions/runs/11856721924/job/33043567199?pr=19835
~~~
The following system packages will first need to be installed:
    pkgconf
~~~

if we want to install pkgconf we must unlink pkg-config
then with `unlink pkg-config; install pkgconf` we get https://github.com/coq/coq/actions/runs/11857007213/job/33044451483
~~~
ERROR while compiling conf-pkg-config.3
# [ERROR] Command not found: pkg-config
~~~

If we install pkgconf but unlink it in favor of pkg-config as in the current code it seems to work

Reply:
Sorry for the breakage, by the way. Even if it would have happened also by releasing a version 4 of the conf package, it may have been clearer that way